I3uckwheat/cards-of-carousal
# Bug/404 socket router errors

#### 0. Make sure your issue is linked:

Close #183 


#### 1. Purpose:


Fixes improperly handled route not found error if an incorrect route is passed to the web socket service.

#### 2. Implementation:


Changes the parameters of what is passed to the `#notFoundHandler` to be what the function is expecting (request and web socket).

#### 3. Possible Issues:




#### 4. How to Test:


1. run `npm dev:backend`
2. try to establish a web socket to a route we do not accept, such as `ws://localhost:4003/test`. You can do this in the dev tools by using `new WebSocket(url)`.
3. observe the web socket request in the network tab(WS tab). A message saying "route-not-found" should be present in the messages.

#### 5. Cleanup
